app/models/comment.rb

```diff
@@ -0,0 +1,8 @@
+class Comment < ApplicationRecord
+  belongs_to :author, class_name: 'User', foreign_key: 'author_id'
+  belongs_to :post
+
+  def update_comments_counter
+    post.increment!(:commentsCounter)
+  end
```

Comment:
- [Optional] COnsider adding `after_save` to call the `update_comments_counter` method every time a new like has been saved to the database, then make the update_comments_counter method private. This will save you the stress of manually updating the comments counter every time a like is added.